src/core/aws-net-websocket-registry.adb

```diff
@@ -466,7 +471,7 @@ package body AWS.Net.WebSocket.Registry is
          Timeout : Duration := Forever;
          Error   : Error_Type := Normal_Closure)
       is
-         W : Object_Class;
+         W : Object_Class := null;
```

Comment:
Why not, but as a pointer it is of course initialized to null already. AWS doesn't seem to have a policy of explicitly initializing pointers ?

Reply:
Sure, that was just to make this explicit as I missed the fact that W could stay null hence the check below. But I agree this change is purely cosmetic.